koku/api/utils.py

```diff
@@ -35,6 +37,29 @@ def get_cost_type(request):
     return cost_type
 
 
+def get_currency(request):
+    """get currency from the DB user settings table or sets currency to default if table is empty."""
+
+    with schema_context(request.user.customer.schema_name):
+        query_settings = UserSettings.objects.all().first()
+        if not query_settings:
+            currency = KOKU_DEFAULT_CURRENCY
+        else:
+            currency = query_settings.settings["currency"]
+    return currency
+
+
+def get_account_settings(request):
+    """Returns users settings from the schema or the default settings"""
+    with schema_context(request.user.customer.schema_name):
+        query_settings = UserSettings.objects.all().first()
+    if not query_settings:
+        user_settings = USER_SETTINGS
+    else:
+        user_settings = query_settings
+    return user_settings
```

Comment:
To make this a tiny bit cleaner you can do the following: 
```suggestion
   if query_settings:
        return query_settings
    return USER_SETTINGS
```